Add Manifest V3 compatibility data for Android (#1750)
mozilla/extension-workshop: src/content/documentation/develop/manifest-v3-migration-guide.md

Manifest V3 became generally available in Firefox 109 after being available as a developer preview from Firefox 101. This page details what's changed and how you adapt your extensions to take advantage of Manifest V3.

> If you are planning to support Android, see the [Developing extensions for Firefox for Android](/documentation/develop/developing-extensions-for-firefox-for-android/#mv3-compatibility) page for additional guidance.

Reviewer: ```suggestion
See the [Developing extensions for Firefox for Android](/documentation/develop/developing-extensions-for-firefox-for-android/#mv3-compatibility) page for additional guidance if you plan to support Firefox for Android.
```

{% capture content %}

> 

Reviewer: ```suggestion
```

<!-- END: Single Column Body Module -->

> 
> 

Reviewer: ```suggestion
```
